Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -460,57 +460,58 @@ If the set point is changed during an autotuning process, a warning will be
 generated. The ongoing tuning process will be halted, and no adjustments will be made
 to the PID parameters.
 </li>
+<li>
+The autotuning must be conducted after the initialization is completed.
+</li>
 </ul>
 <h4>Guidance for setting the parameters</h4>
 <p>
-The performance of the autotuning is affected by the parameters including the
+The performance of the autotuning is determined by several parameters, including the
 typical range of the control error <code>r</code>,
 the reference output for the tuning process <code>yRef</code>, the higher and
 lower values for the relay output <code>yHig</code> and <code>yLow</code>, and the
-deadband <code>deaBan</code>. These parameters can be specified with the following
-steps:
+deadband <code>deaBan</code>.
+Generally, these parameters must be manually specified by users on a case-by-case basis. 
```

Comment:
don't use `Generally`, as the user may ask "what is not in general?" Also, there is no need to include "manually". In Modelica, parameters are specified by user. So I suggest change sentence as below:

`These parameters must be specified on a case-by-case basis. To set them, the user should conduct following steps.`